internal/chart/procfile.go

```diff
@@ -101,3 +103,18 @@ func routableProcess(names []string) string {
 	sort.Strings(names)
 	return names[0]
 }
+
+func WriteProcfile(processes []ketchv1.ProcessSpec, dest string) error {
+	f, err := os.Create(dest)
```

Comment:
if there is a user's Procfile in the directory, we overwrite it and replace its content with processes defined in `app.yaml`. 
Not sure that it's OK

Reply:
Added logic to check for existing Procfile and error when deploying from source via yaml. Also added logic to clean up generated Procfiles. Feels a little dirty, but I don't have a better solution. github.com/buildpacks/pack seems to depend on a Procfile existing at project root, as far as I can tell.